lighthouse-core/lib/sentry.js

```diff
@@ -44,7 +50,19 @@ sentryDelegate.init = function init(opts) {
 
     // Special case captureException to return a Promise so we don't process.exit too early
     sentryDelegate.captureException = (...args) => {
-      if (args[0] && args[0].expected) return Promise.resolve();
+      const empty = Promise.resolve();
+      // Ignore expected errors
+      if (args[0] && args[0].expected) return empty;
+      // Ignore errors matching our shortlist
+      if (args[0] && IGNORED_ERRORS.some(regex => regex.test(args[0].message))) return empty;
+      // Only report 25% of production errors
+      if (!isDevelopment && Math.random() > 0.25) return empty;
```

Comment:
we have a fairly long tail of errors reported. and i'm seeing quite a few with < 3 reports that look worthwhile.  this makes me hesitant to throw so many away.

Reply:
could we sample just the 'info' level events?

Reply:
> could we sample just the 'info' level events?

yes good idea, but FWIW nothing is actually info yet :)

> we have a fairly long tail of errors reported. and i'm seeing quite a few with < 3 reports that look worthwhile. this makes me hesitant to throw so many away.

true, I might argue that given our current outstanding issues if something is <0.1% right now we likely shouldn't be investing time to fix it

seems like we need to pick at least 1 of the following:
* ignore more known issues (like falling out of memory cache)
* sample events
* upgrade our plan